src/app/home/home.module.ts

```diff
@@ -1,3 +1,4 @@
+import { SpaceWizardModule } from '../space-wizard/space-wizard.module';
```

Comment:
Unfortunately tools don't seem to recognise this so it's too easy to get it wrong. Maybe we just drop this styleguide until tools can support it?